net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -18,6 +18,11 @@
                 <pubkey type="TextField">
                     <Required>N</Required>
                 </pubkey>
+                <psk type="TextField">
+                    <Required>N</Required>
+                    <mask>/^[a-z0-9\=\/]{44}$/i</mask>
+                    <ValidationMessage>Should be a string of 44 characters and ending with = like a public key. Allowed characters are 0-9a-zA-Z/=</ValidationMessage>
```

Comment:
Assuming this is Base64, the "=" is used for padding and is not entirely mandatory. Whitespaces and line breaks are normally allowed and ignored. It would also mean "+" is missing from your validation.